README.md

```diff
@@ -1,8 +1,43 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Braikhna Yousafzai . Jarvis Consulting
 
+I enjoy the growth and constant learning that comes with the software industry. I am looking to apply and grow my development and engineering skills in creative ways. I enjoy working collaboratively and across disciplines to get a comprehensive understanding of my projects and company. I appreciate team-based approaches towards a cohesive solution. I am a recent graduate from Western University with a Bachelor's degree with Specialization in Computer Science. I have worked on several Java, Python, and database management projects throughout my degree. I most enjoy the engineering, data management, and analytic aspects of the industry. Outside of work, I enjoy painting, coffee, and a good book. I am also quite involved with my community in volunteering and leadership capacities.
+
+## Skills
+
+**Proficient:** Java, RDBMS/SQL, Agile/Scrum, Linux/Bash, Git, C/C++
+
+**Competent:** Python, R, Docker, PHP, HTML, CSS
+
+**Familiar:** Hadoop, NoSQL, Tableau, Crystal Reports, Scala
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai](https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai/tree/master/linux_sql)]: Implemented a linux cluster monitoring agent using shell scripts inserting hardware and usage data for each cluster node to a database. The data was collected in real time using a Crontab job and persisted using PostgreSQL. The project was deployed using a docker container and version control was maintained through GitHub.
```

Comment:
fix the URI
Implemented a Linux cluster monitoring agent using Shell scripts inserting hardware usage data for each cluster node into a PostgreSQL database. The data was collected in real-time using a crontab job and persisted using PostgreSQL. The project was deployed using a docker container, and version control was maintained through GitHub.

SQL and analytics